source/settings.json

```diff
@@ -156,6 +186,26 @@
       "label": "Welcome text",
       "default": "#121212"
     },
+    {
+      "variable": "badge_text_color_primary",
+      "label": "Badge text color (primary)",
+      "default": "#222222"
+    },
+    {
+      "variable": "badge_background_color_primary",
+      "label": "Badge background color (primary)",
+      "default": "#FFD700"
+    },
+    {
+      "variable": "badge_text_color_secondary",
+      "label": "Badge text color (secondary)",
+      "default": "#222222"
+    },
+    {
+      "variable": "badge_background_color_secondary",
+      "label": "Badge background color (secondary)",
+      "default": "#B7B7B7"
+    },
```

Comment:
Do we for sure want to include "color" in all of these labels?

Reply:
@nickendle updated to remove all the color labels as discussed